packages/react-native-executorch/common/rnexecutorch/models/vertical_ocr/VerticalDetector.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <executorch/extension/tensor/tensor_ptr.h>
+#include <opencv2/opencv.hpp>
+
+#include <rnexecutorch/models/BaseModel.h>
+#include <rnexecutorch/models/ocr/Types.h>
+
+namespace rnexecutorch {
+using executorch::aten::Tensor;
+using executorch::extension::TensorPtr;
+
+class VerticalDetector : public BaseModel {
+public:
+  VerticalDetector(const std::string &modelSource, bool detectSingleCharacters,
+                   std::shared_ptr<react::CallInvoker> callInvoker);
+  std::vector<DetectorBBox> generate(const cv::Mat &inputImage);
+  cv::Size getModelImageSize() const noexcept;
```

Comment:
And again, make sure that generate can be marked with `noexcept`
```suggestion
class VerticalDetector final : public BaseModel {
public:
  explicit VerticalDetector(const std::string &modelSource, bool detectSingleCharacters,
                   std::shared_ptr<react::CallInvoker> callInvoker);
  std::vector<DetectorBBox> generate(const cv::Mat &inputImage);
  cv::Size getModelImageSize() const noexcept;
```